instrumentation.go

```diff
@@ -66,16 +105,15 @@ func InstrumentAndLoadCollection(
 		if err != nil {
 			const ENOSPC = syscall.Errno(0x1c)
```

Comment:
Note: `VerifierError.Truncated` can now be used reliably for checking log truncation, no need to check ENOSPC any longer.